plaso/parsers/dsv_parser.py

```diff
@@ -111,10 +106,9 @@ def _CreateDictReader(self, line_reader):
     quotechar = self.QUOTE_CHAR
     magic_test_string = self._MAGIC_TEST_STRING
     # Python 3 csv module requires arguments to constructor to be of type str.
```

Comment:
Can remove this comment

Reply:
ack more clean up of the DSV parser in https://github.com/log2timeline/plaso/pull/2935